### DISCUSSION TITLE
optimal settings for low-complexity samples (immunoaffinity enriched)

### DISCUSSION THREAD

**Discussion by MichelMoser** [vdemichev/DiaNN#813] [General]
Hi, 
I am new to proteomics and would like to discuss DIA-NN parameter settings for low-complexity protein samples. 

An example of a sample would be antibody-enriched protein. 
I would expect to see the target protein with its isoforms, antibody, and BSA (from the beads). 
As the protein database for those candidates is very small (5 proteins) it gets difficult to set a sensible FDR. 

Do you have experience with how to set FDR in such a scenario?
Or should other parameters be tuned as well?  

Would it perhaps be better to still use a complex database to create a sensible library/decoy-library for FDR estimation? 

Of course, running DDA might be another path to take, but since DIA sounds like a very sensitive and reproducible approach to quantifying low-abundant peptides, I would very much like to use DIA-NN. 

Thank you, Michel

**Comment by vdemichev** [comment]
Hi Michel,

One thing that definitely makes sense in this case is to include also the whole database of the organism in question. FDR should be fine. Filtering the output using PEP (posterior error probability, present in the main DIA-NN report) in addition to q-values can be helpful to gain extra confidence in the subset of the proteins you are interested in.

Best,
Vadim
